frontend-new/src/feedback/feedbackForm/components/customRating/CustomRating.tsx

```diff
@@ -57,27 +59,21 @@ const CustomRating: React.FC<CustomRatingProps> = ({
       gap={theme.spacing(2)}
       data-testid={DATA_TEST_ID.CUSTOM_RATING_CONTAINER}
     >
-      <Box display="flex" flexDirection="column" gap={theme.spacing(1)}>
-        <Typography
-          variant="subtitle1"
-          color={(theme) => theme.palette.text.secondary}
-          data-testid={DATA_TEST_ID.CUSTOM_RATING_TEXT}
-        >
-          {questionText}
-        </Typography>
+      <Box display="flex" flexDirection="column" gap={theme.tabiyaSpacing.sm}>
+        <QuestionText questionText={questionText} />
         {displayRating && (
           <Box display="flex" flexDirection="column" width="fit-content">
             <Rating
               name={questionId}
               value={ratingValue}
               onChange={(_, newValue) => handleRatingChange(newValue)}
-              max={10}
+              max={maxRating}
               precision={1}
               size="small"
```

Comment:
The rating overflows horizontally on mobile devices. It requires horizontal scrolling which we want to avoid

Reply:
![image](https://github.com/user-attachments/assets/a20783b1-bc43-47e6-b54a-9d55de46adf9)